src/main.js

```diff
@@ -179,11 +180,24 @@ function goToMain() {
   hide(savedPoster)
 };
 
-/*
-passing imageURL, title, and quote into a new Poster
-have those be randomly pulled from the arrays
-inject Poster.value into the DOM at the correct places
+// event.preventDefault()
 
-on button click generate a new Poster object and inject those values into the page.
+function saveThisPoster() {
+  //click “Save This Poster” button
```

Comment:
We should probably delete these comment lines to make the function read cleaner when refactoring.